goosebit/updater/routes.py

```diff
@@ -1,29 +1,21 @@
 import time
 
-from fastapi import APIRouter, Depends, HTTPException
+from fastapi import APIRouter, Depends
 from fastapi.requests import Request
 
-from goosebit.settings import config
-
 from . import controller
 from .manager import get_update_manager
 
 
-async def verify_tenant(tenant: str):
-    if not tenant == config.tenant:
-        raise HTTPException(404)
-    return tenant
-
-
 async def log_last_connection(request: Request, dev_id: str):
     host = request.client.host
     updater = await get_update_manager(dev_id)
     await updater.update_last_connection(round(time.time()), host)
 
 
 router = APIRouter(
-    prefix="/{tenant}",
-    dependencies=[Depends(verify_tenant), Depends(log_last_connection)],
+    prefix="/ddi",
```

Comment:
I suppose in the future we may want to have this just be a variable, then we can introduce `tenant` as another firmware selection layer?  Not sure if it will be better than what we have with device IDs and rollouts, but maybe it will allow automating the initial selection of those variables?

Reply:
IMHO it would be fine to just change the default to „ddi“. The advantage of leaving (configurable) tenant support would be that devices currently connected to hawkBit would not have to be changed for gooseBit.

Reply:
Overlooked this comment before merging - sorry. Still would strive for simplicity by hard-coding it.